.env.example

```
@@ -0,0 +1,61 @@
+EMAIL_USE_TLS=True
+EMAIL_HOST_USER=your-email@example.com
+EMAIL_HOST_PASSWORD=your-email-password
+DEFAULT_FROM_EMAIL=noreply@example.com
+
+# Sentry Configuration (optional - for error tracking)
+SENTRY_DSN=
+
+# Security Headers
+SECURE_SSL_REDIRECT=True
+SESSION_COOKIE_SECURE=True
+CSRF_COOKIE_SECURE=True
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CSRF_COOKIE_SECURE key should go before the SECURE_SSL_REDIRECT key

(UnorderedKey)